cleanlab/datalab/datalab.py

```diff
@@ -635,3 +636,64 @@ def load(path: str, data: Optional[Dataset] = None) -> "Datalab":
         load_message = f"Datalab loaded from folder: {path}"
         print(load_message)
         return datalab
+
+    def _spurious_correlation(
+        self, properties_of_interest: Optional[List[str]] = None
```

Comment:
For now, let's omit this argument in `Datalab._spurious_correlation()`.

Remember to remove the parameter in the docstring as well.